### ISSUE TITLE
Seed for XGBoost for Random Forest in section 8.3 

### ISSUE THREAD

**Issue by fizban99** [mattharrison/effective_xgboost_book#7]
It seems that using the seed 42 with xgboost 1.7.6 with the sample data used in section 8.3 returns a score of 
`0.7535911602209945`
which is different from the value on the book. Actually, using the value 39 for the seed replicates the value of the book of
`0.7447513812154696`
The default hyperparameters are mostly set to None in 1.7.6, but that does not seem to affect the score.
